source/macos/AdaptiveCards/AdaptiveCards/Views/ACRContentStackView.swift

```diff
@@ -309,6 +309,10 @@ extension ACRContentStackView: InputHandlingViewErrorDelegate {
         hideErrorMessage(with: currentFocussedView)
     }
     
+    func inputHandlingViewShouldAnnounceErrorMessage() {
+        NSAccessibility.announce("Error! " + (errorMessageField?.stringValue ?? "") + ". " + (inputLabelField?.stringValue ?? ""))
+    }
```

Comment:
Is this tested?
Just curious how is "Error '!'" is read. Does it explicitly say "exclamation" or just read Error?

Reply:
It just says Error